src/Check/DiskFree.php

```diff
@@ -31,6 +31,8 @@
  *     authors:   Dave Ingram <dave@dmi.me.uk>, Nick Pope <nick@nickpope.me.uk>
  *     license:   http://creativecommons.org/licenses/BSD/ CC-BSD
  *     copyright: Copyright (c) 2010, Dave Ingram, Nick Pope
+ *
+ * @psalm-type ResultData array{availability: array{value: float, valueType: string}}
  */
 class DiskFree extends AbstractCheck implements CheckInterface
```

Comment:
That means that other code repositories with custom implementations of the interface, would also suddenly have failing psalm/phpstan checks right?
It would be the better solution yes, and mean that all new checks would also set this up.
If the static-analysis issue is okay (don't know if it falls under a bc break, as the code api doesnt change) I will change this :)

Reply:
@Ocramius
Did a small test, and the current psalm version (4.30) does not error when I add a template to CheckInterface. Updating locally to psalm 5.6 does trigger `MissingTemplateParam` errors.

Reply:
I would leave #62 alone for now :) 

As for the patch: please rebase, instead of merging upstream.